node/src/node_configurator/configurator.rs

```diff
@@ -59,15 +61,17 @@ impl Handler<BindMessage> for Configurator {
     type Result = ();
 
     fn handle(&mut self, msg: BindMessage, _ctx: &mut Self::Context) -> Self::Result {
-        self.node_to_ui_sub = Some(msg.peer_actors.ui_gateway.node_to_ui_message_sub.clone());
-        self.new_password_subs = Some(vec![msg.peer_actors.neighborhood.new_password_sub])
+        self.node_to_ui_sub_opt = Some(msg.peer_actors.ui_gateway.node_to_ui_message_sub.clone());
+        self.configuration_change_msg_sub_opt =
+            Some(msg.peer_actors.neighborhood.configuration_change_msg_sub);
     }
 }
 
 impl Handler<NodeFromUiMessage> for Configurator {
     type Result = ();
 
     fn handle(&mut self, msg: NodeFromUiMessage, _ctx: &mut Self::Context) -> Self::Result {
+        // TODO: I wish if we would log the body and context_id of each request over here. Is there a security risk?
```

Comment:
There isn't a security risk in logging the context ID. There's definitely a security risk in logging the new password. There's a small security risk in logging the old password, but it's not very big. (Suppose you saw a series of logged password changes: password100 -> ?, password101 -> ?, password102 -> ?. What do you think the password is now?)

For forensic purposes, logging the context ID and the old password is probably fine.

Reply:
Done 👍 

I see. This TODO statement was just a part of curiosity. The logs that I intend to add for this card were added at the separate place and I don't think that I feel the need to put the log over here for now. Thanks for the explanation.